openmeter/sink/storage.go

```diff
@@ -58,14 +59,30 @@ func (q InsertEventsQuery) ToSQL() (string, []interface{}, error) {
 
 	query := sqlbuilder.ClickHouse.NewInsertBuilder()
 	query.InsertInto(tableName)
-	query.Cols("namespace", "validation_error", "id", "type", "source", "subject", "time", "data")
+	query.Cols("namespace", "validation_error", "id", "type", "source", "subject", "time", "data", "ingested_at", "created_at")
 
 	for _, message := range q.Messages {
 		var eventErr string
 		if message.Status.Error != nil {
 			eventErr = message.Status.Error.Error()
 		}
 
+		ingestedAt := time.Now()
+		createdAt := time.Now()
```

Comment:
```suggestion
		createdAt := time.Now()
		ingestedAt := createdAt
```

Time is immutable, it's safe to copy it and it makes sure the monotonic clock remains the same.